test/e2e/features/recover.feature

```diff
@@ -0,0 +1,117 @@
+Feature: Recover cluster extension from errors that might occur during its lifetime
+
+  Background:
+    Given OLM is available
+    And "test" catalog serves bundles
+
+
+  Scenario: Restore removed resource
+    Given service account "olm-sa" with needed permissions is available in test namespace
+    And ClusterExtension is applied
+      """
+      apiVersion: olm.operatorframework.io/v1
+      kind: ClusterExtension
+      metadata:
+        name: $NAME
+      spec:
+        namespace: $TEST_NAMESPACE
+        serviceAccount:
+          name: olm-sa
+        source:
+          sourceType: Catalog
+          catalog:
+            packageName: test
+            selector:
+              matchLabels:
+                "olm.operatorframework.io/metadata.name": test-catalog
+      """
+    And ClusterExtension is available
+    And resource "configmap/test-configmap" is available
+    When resource "configmap/test-configmap" is removed
+    Then resource "configmap/test-configmap" is eventually restored
+
+  Scenario: Install ClusterExtension after target namespace becomes available
+    Given ClusterExtension is applied
+      """
+      apiVersion: olm.operatorframework.io/v1
+      kind: ClusterExtension
+      metadata:
+        name: $NAME
+      spec:
+        namespace: $TEST_NAMESPACE
+        serviceAccount:
+          name: olm-sa
+        source:
+          sourceType: Catalog
+          catalog:
+            packageName: test
+            selector:
+              matchLabels:
+                "olm.operatorframework.io/metadata.name": test-catalog
+      """
+    And ClusterExtension reports Progressing as True with Reason Retrying
+    When service account "olm-sa" with needed permissions is available in test namespace
```

Comment:
I wonder if it should be:

```suggestion
    When service account "olm-sa" with needed permissions is available in $TEST_NAMESPACE
```

To make it (extra) clear those two things are the same..?